react/lib/components/Widget/WidgetContainer.tsx

```diff
@@ -132,32 +133,29 @@ export const WidgetContainer: React.FunctionComponent<WidgetContainerProps> =
 
     const handlePayment = useCallback(
       (transaction: Transaction) => {
-        if (sound && !hideToasts) txSound.play().catch(() => {});
-
-        const {
-          amount: transactionAmount,
-          paymentId: transactionPaymentId } = transaction;
-        const receivedAmount = resolveNumber(transactionAmount);
-
-        const currencyTicker = getCurrencyTypeFromAddress(to);
-        if (!hideToasts)
-          enqueueSnackbar(
-            `${
-              successText ? successText + ' | ' : ''
-            }Received ${receivedAmount} ${currencyTicker}`,
-            snackbarOptions,
-          );
-        const txPaymentId = transactionPaymentId
-        const isCryptoAmountValid = (cryptoAmount && receivedAmount.isEqualTo(resolveNumber(cryptoAmount))) || !cryptoAmount;
-        const isPaymentIdValid = thisPaymentId ? txPaymentId === thisPaymentId : true;
-
-        if (isCryptoAmountValid && isPaymentIdValid)
-        {
+        const { amount } = transaction;
```

Comment:
I don't this this deconstruction adds clarity. This will only be used once below, and in a context where there are two amounts: the expected (named `cryptoAmount`) and the received `transaction.amount`. Not adding this line and using `transaction.amount` below makes more clear that the value being used there is the one received in the transaction.